Check-Substring.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
```

Comment:
Try to add questions so that it will be easy to understand what the program does